sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosE2EOperationRetryPolicyConfigBuilder.java

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos;
+
+import com.azure.cosmos.implementation.OperationCancelledException;
+
+import java.time.Duration;
+
+/**
+ * Builder for CosmosEndToEndOperationLatencyPolicyConfig
+ */
+public class CosmosE2EOperationRetryPolicyConfigBuilder {
```

Comment:
Should also be final?

Reply:
The other two builders CosmosClientBuilder and ChangeFeedProcessorBuilder are not final